src/common/transformations/src/transformations/common_optimizations/convert_compressed_to_mixed_precision.cpp

```diff
@@ -7,21 +7,21 @@
 #include "itt.hpp"
 #include "openvino/opsets/opset8.hpp"
 #include "openvino/pass/manager.hpp"
+#include "transformations/common_optimizations/align_mixed_fp32_fp16_types.hpp"
 #include "transformations/common_optimizations/convert_compression_only_to_legacy.hpp"
 #include "transformations/common_optimizations/mark_precision_sensitive_shapeof_subgraphs.hpp"
+#include "transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp"
 #include "transformations/convert_precision.hpp"
 #include "transformations/utils/utils.hpp"
 
 bool ov::pass::ConvertCompressedToMixedPrecision::run_on_model(const std::shared_ptr<ov::Model>& f) {
     RUN_ON_MODEL_SCOPE(ConvertCompressedToMixedPrecision);
 
-    // pass is triggered only for fp16 compressed Models
-    if (!ov::op::util::has_decompression_converts(f))
-        return false;
-
     Manager manager(get_pass_config());
     // Mark nodes in ShapeOf subgraphs with disable_fp16_compression rt_info to keep them in FP32 precision
     REGISTER_PASS(manager, MarkPrecisionSensitiveShapeOfSubgraphs)
+    REGISTER_PASS(manager, MarkSugraphsToKeepInMixedPrecision)
```

Comment:
I think these two passes should be combined. `MarkPrecisionSensitiveShapeOfSubgraphs)` should be a sort of container incorporating several transformations marking such sub-graphs

Reply:
`MarkSugraphsToKeepInMixedPrecision` is a container which contains several markings I think we can move `MarkPrecisionSensitiveShapeOfSubgraphs` inside `MarkSugraphsToKeepInMixedPrecision`. Do you agree with that?